app/src/test/java/org/mozilla/fenix/ext/BookmarkNodeTest.kt

```diff
@@ -0,0 +1,41 @@
+package org.mozilla.fenix.ext
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.fenix.TestApplication
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import mozilla.components.concept.storage.BookmarkNode
+import mozilla.components.concept.storage.BookmarkNodeType
+
+@RunWith(RobolectricTestRunner::class)
+@Config(application = TestApplication::class)
+class BookmarkNodeTest {
+
+    @Test
+    fun `minus`() {
```

Comment:
nit: I know this is common on the team but I don't like this test naming scheme – i.e. `minus`. It should describe what it's doing. I really like the [GIVEN WHEN THEN](https://martinfowler.com/bliki/GivenWhenThen.html) that we used on Fire TV: GIVEN <some initial app state> WHEN <some event happens> THEN <the expected result>. It encourages the tester to think through what _this specific test_ is testing and thus what this specific test is _not_ testing. Then the tester might cover a few more classes of problem (see my above comment).